### PR TITLE
grub-mkconfig: add Spanish translation

### PR DESCRIPTION
**Pull request by mag4no10**
- [ x ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ x ] The page description(s) have links to documentation or a homepage.
- [ x ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ x ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/grub-mkconfig.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# grub-mkconfig
+
+> Generar un archivo de configuracion de GRUB.
+> Para mas informacion: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>.
```
Comment (blueskyson) [inline pages.es/linux/grub-mkconfig.md:4]: We have translations for [more information](https://github.com/tldr-pages/tldr/blob/6d01f29bbd9135553d255574197f293d2942885a/scripts/set-more-info-link.py#L18).
```suggestion
> Más información: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkconfig.html>.
```

**Comment by mag4no10** [comment]
May I close the push request?

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@mag4no10 Any reason why this PR was closed? 👀

**Comment by mag4no10** [comment]
@kbdharun isn't the PR finished? If not I apologize. Ok I will stay tuned.

**Comment by kbdharun** [comment]
> @kbdharun isn't the PR finished? If not I apologize. Ok I will stay tuned.

No issues, A PR is complete it is merged. After which you can delete the branch.

When a change is no longer needed at that time you can choose to close the PR.

**Pull request merged**